.prettierrc.cjs

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  trailingComma: 'es5',
+  tabWidth: 2,
```

Comment:
тут стоит дописать editorconfig: true

Reply:
@nosovk схоже, такої опції prettierrc не існує
"editorconfig: true is not a prettierrc option, it's an option when using the prettier API from another package. .... From the prettier side, they say it will always use an .editorconfig file if found."
[https://github.com/gadicc/node-yahoo-finance2/issues/56](https://github.com/gadicc/node-yahoo-finance2/issues/56)

![photo_2022-10-27_09-41-13](https://user-images.githubusercontent.com/76002690/198213891-3f39abcb-f90f-4b75-8b12-e9b1cc3e1807.jpg)
![photo_2022-10-27_09-42-23](https://user-images.githubusercontent.com/76002690/198213898-750ff31b-a06f-4e01-a033-0b5c1c05a327.jpg)

Reply:
ok